packages/grafana-ui/src/utils/dom.ts

```diff
@@ -30,12 +30,12 @@ export function getPreviousCousin(node, selector) {
 
 export function getNextCharacter(global = window) {
   const selection = global.getSelection();
-  if (!selection.anchorNode) {
+  if (!selection || !selection.anchorNode) {
     return null;
   }
 
   const range = selection.getRangeAt(0);
   const text = selection.anchorNode.textContent;
   const offset = range.startOffset;
-  return text.substr(offset, 1);
+  return text!.substr(offset, 1);
```

Comment:
What magic is this?

Reply:
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator

Reply:
It is essentially a way to say "the types think it could be null... but lets behave like it will never be null"